backend/opencv_backend.py

```diff
@@ -0,0 +1,49 @@
+import cv2
+
+from backend import Backend, PublishFunc
+from overlay import Canvas
+
+class OpenCVBackend(Backend):
+    """ Gets and displays video using the OpenCV (`cv2`) library.
+
+        This is intended for use with laptops with webcams, not on the
+        Raspberry Pi. """
+
+    def __init__(self, width: int, height: int, publish_recording_status_func: PublishFunc):
+        super().__init__(width, height, publish_recording_status_func)
+
+        self.webcam = None
+
+        # Time between video frames when running on OpenCV, in milliseconds
+        self.frametime = 17
```

Comment:
Any reason why this is exactly 17?

Reply:
The frame time when running at 60 fps is around 16.77 ms. I've made that more clear in the code. I'm not sure if it has to be an integer or a float (and can't test), so I've left it as a float for now, let me know if it gets unhappy. (In the C version of OpenCV it has to be an integer)

Reply:
Would be good to mention that this value is from 60fps because I didn't know that! Maybe reword your comment?

Reply:
Yeah definitely, sorry I would have marked it as resolved but hadn't pushed the change yet whoops